quinn-udp/src/unix.rs

```diff
@@ -330,7 +339,7 @@ fn send(
             &mut hdr,
             &mut iov,
             &mut ctrl,
-            false,
+            encode_srcip,
```

Comment:
nit: pass `cfg!(target_os = "macos")` directly here and we don't need to bother defining and mutating a separate `encode_srcip` variable.